sign-in-with-esignet/src/lib/SignInWithEsignet/SignInWithEsignet.test.ts

```diff
@@ -0,0 +1,126 @@
+import init from "./SignInWithEsignet";
+
+describe(
+  "init", () => {
+    const oidcConfig = {
+      authorizeUri: "https://example.com/authorize",
+      redirect_uri: "https://example.com/callback",
+      client_id: "client123",
+      scope: "openid"
+    };
+
+    const buttonConfig = {
+      theme: "outline",
+      shape: "sharp_edges",
+      width: '150px',
+      background: '#3498db', // Background color of the button (CSS style)
+      textColor: '#ffffff',
+      borderColor: '#2980b9', // Border color of the button (CSS style)
+      borderWidth: '2px', // Border width of the button (CSS style)
+      font: '16px Arial', // Font style for the button (CSS style)
+      fontFamily: 'Arial, sans-serif',
+    }
+
+    const signInElement = document.createElement('div');
+
+    const style = {
+      backgroundColor: 'red',
+    };
+
+    beforeEach(() => {
+        signInElement.innerHTML = '';
+    });
+
+    it("should return 'Required parameter is missing' when required parameters are missing", () => {
+      const invalidOidcConfig = {
+        authorizeUri: "https://example.com/authorize",
```

Comment:
test cases for other required parameters can be added as well. Name the testcase accordingly